config/flags_node.go

```diff
@@ -108,7 +108,7 @@ var (
 	FlagShaperBandwidth = cli.Uint64Flag{
 		Name:  "shaper.bandwidth",
 		Usage: "Set the bandwidth limit in bytes",
-		Value: 5000,
+		Value: 5_000_000,
```

Comment:
This value is set in Kbytes based on the code that uses it. 
So the description is not correct, not the value. 
If you want to change it to real bytes underlying code needs to be changed.

Reply:
I see, I'll change the description then